numpy/ma/tests/test_core.py

```diff
@@ -2853,39 +2853,51 @@ def test_inplace_multiplication_array_type(self):
 
     def test_inplace_floor_division_scalar_type(self):
         # Test of inplace division
+        # Check for TypeError in case of unsupported types
+        unsupported = {np.complex64, np.complex128, np.complex256}
```

Comment:
Ohh seems like `complex256` is not there on all architectures. Will fix this.

Reply:
You can try iterating over `np.typecodes["Complex"]`, which will yield the platform-agnostic type-codes of all complex types.

Reply:
Thanks! It worked